package.json

```diff
@@ -114,16 +114,16 @@
     "enzyme": "^3.4.3",
     "enzyme-adapter-react-16": "^1.2.0",
     "enzyme-to-json": "^3.3.4",
-    "eslint": "4",
-    "eslint-config-prettier": "^2.9.0",
+    "eslint": "^5.4.0",
```

Comment:
Last I checked, this version of eslint still had issues. I think we should hold off on this one for a bit more... Thoughts?

Reply:
i expected to issues. do you have any links?